qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,37 @@
+- hotplug_virtio_mem:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    aarch64:
+        smp = 8
+        vcpu_maxcpus = ${smp}
```

Comment:
```suggestion
```

Reply:
@PaulYuuu why are you suggesting to delete this block since vcpu_maxcpus is needed for aarch64?

Reply:
By default, they are the same actually. Either upstream or internal will pick up one of them as the main to set the smp value, I don't know why you guys will talk about this if we want to delete CPU related parameters.

Reply:
Got it, thanks for the explanation

Reply:
Changed now, thanks!